keyboards/tanuki/README.md

```diff
@@ -0,0 +1,8 @@
+# Tanuki 40% keyboard
```

Comment:
This file should be lowercase.  You may need to rename it to something else, commit, and rename it to readme.md to get it to take, properly. 

Also, could you include the maintainer, support and availability entries, as well? Even if they're not links.